hedera-mirror-web3/src/main/java/com/hedera/services/store/models/TokenRelationship.java

```diff
@@ -80,10 +84,11 @@ public TokenRelationship(
         this.destroyed = destroyed;
         this.notYetPersisted = notYetPersisted;
         this.automaticAssociation = automaticAssociation;
+        this.hasAssociation = hasAssociation;
         this.balanceChange = balanceChange;
     }
 
-    public TokenRelationship(Token token, Account account) {
+    public TokenRelationship(Token token, Account account, Boolean hasAssociation) {
```

Comment:
Mismatch between field and parameter.
```suggestion
    public TokenRelationship(Token token, Account account, boolean hasAssociation) {
```